Assets/HoloToolkit/SpatialMapping/Scripts/RemoteMapping/SimpleMeshSerializer.cs

```diff
@@ -50,6 +50,34 @@ public static byte[] Serialize(IEnumerable<Mesh> meshes)
         }
 
         /// <summary>
+        /// Serializes a list of MeshFilter objects into a byte array.
+        /// Transforms vertices into world space before writing to the file.
+        /// </summary>
+        /// <param name="meshes">List of MeshFilter objects to be serialized.</param>
+        /// <returns>Binary representation of the Mesh objects.</returns>
+        public static byte[] Serialize(IEnumerable<MeshFilter> meshes)
+        {
+            byte[] data = null;
```

Comment:
It's a good idea not to initialize variables to invalid values in C#.  That way, the compiler will catch you at compile-time if you accidentally forget to set it before using it.  I suggest removing the ` = null` here.